common/src/main/res/layout/dialog_adaptive.xml

```diff
@@ -37,7 +37,8 @@
         android:layout_height="wrap_content"
         android:layout_marginTop="20dp"
         android:gravity="center"
-        android:textAlignment="gravity"
+        android:layout_gravity="center"
+        android:textAlignment="center"
```

Comment:
This breaks the logic of aligning a message that is longer than 3 lines to the left (right in rtl locales). Is there a reason for this change?

Reply:
@Syn-McJ I tried to match new figma design 
<img width="609" alt="image" src="https://user-images.githubusercontent.com/1075310/167405936-42b11ec1-1989-45a6-ba16-76b38d9d7635.png">

Reply:
You should be able to achieve that just by removing the icon though? Changing alignment logic isn't necessary.